server.go

```diff
@@ -69,7 +69,7 @@ func (s *QProxyServer) ListQueues(in *rpc.ListQueuesRequest, stream rpc.QProxy_L
 	defer func() {
 		s.m.APILatency.WithLabelValues("ListQueues", in.Namespace, "").Observe(float64(time.Now().Sub(start)))
 		if err != nil {
-			s.m.APIErrors.WithLabelValues("ListQueues", in.Namespace, "").Inc()
+			s.m.APIErrors.WithLabelValues("ListQueues", in.Namespace, "", err.Error()).Inc()
```

Comment:
what does err.Error() look like? Is it a full sentence or just a single word? 
We can't allow full sentences as label as each different string will create a new time series. So it should be somewhat like an enum with a small number of different values.